modules/currency/currency.service.js

```diff
@@ -12,17 +12,18 @@ const currencyErrorMessage = JSON.stringify([
 ]);
 class CurrencyService {
   async getAllCurrencies() {
-    return (await Currency.find()) || new Error(currencyErrorMessage);
+    const currency = await Currency.find();
+    return currency;
   }
 
   async getCurrencyById(id) {
-    return (await Currency.findById(id)) || new Error(currencyErrorMessage);
+    return (await Currency.findById(id)) || new Error(CURRENCY_NOT_FOUND);
   }
 
   async updateCurrency(id, currency) {
     return (
       (await Currency.findByIdAndUpdate(id, currency))
```

Comment:
Please, add third parameter ` {new: true}` to return already updated entity

Reply:
fixed